js/symbol/shaping.js

```diff
@@ -146,8 +146,8 @@ function justifyLine(positionedGlyphs, glyphs, start, end, justify) {
 
 }
 
-function align(positionedGlyphs, justify, horizontalAlign, verticalAlign, maxLineLength, lineHeight, line) {
-    var shiftX = (justify - horizontalAlign) * maxLineLength;
+function align(positionedGlyphs, justify, horizontalAlign, verticalAlign, maxLineLength, lineHeight, line, translate) {
+    var shiftX = (justify - horizontalAlign) * maxLineLength + translate[0];
```

Comment:
vertical translation is still handled on line 36. It might be clearer to move it down here though